SCons/Tool/ninja/ninja.xml

```diff
@@ -291,37 +307,38 @@ See its __doc__ string for a discussion of the format.
     <cvar name="NINJA_FILE_NAME">
         <summary>
             <para>
-                The filename for the generated Ninja build file defaults to
-                <filename>ninja.build</filename>
+                The filename for the generated Ninja build file.
+                The default is <filename>ninja.build</filename>.
             </para>
         </summary>
     </cvar>
 
     <cvar name="NINJA_ALIAS_NAME">
         <summary>
             <para>
-                Name of the Alias() which is will cause SCons to create the <filename>ninja.build</filename> file, and
-                then (optionally) run ninja.
+                The name of the alias target which will cause &SCons; to create the &ninja; build file,
+                and then (optionally) run &ninja;.
+                The default value is <literal>generate-ninja</literal>.
             </para>
         </summary>
     </cvar>
 
     <cvar name="NINJA_SYNTAX">
         <summary>
             <para>
-                Theres also NINJA_SYNTAX which is the path to a custom ninja_syntax.py file which is used in generation.
-                The tool currently assumes you have ninja installed through pip, and grabs the syntax file from that
-                installation if none specified.
+                The path to a custom <filename>ninja_syntax.py</filename> file which is used in generation.
+                The tool currently assumes you have &ninja; installed as a &Python; module and grabs the syntax file from that
+                installation if &cv-NINJA_SYNTAX; is not explicitly set.
             </para>
         </summary>
     </cvar>
 
     <cvar name="NINJA_FORCE_SCONS_BUILD">
         <summary>
             <para>
-                When NINJA_FORCE_SCONS_BUILD is True, this will cause the build nodes to callback to scons instead of using
-                ninja to build them. This is intended to be passed to the environment on the builder invocation.
-                It is useful if you have a build node which does something which is not easily translated into ninja.
+                If true, causes the build nodes to callback to scons instead of using
```

Comment:
True instead of true?

Reply:
Was trying to get across truthiness, not the specific constant value True.